tests/simulation/distributions_test.py

```diff
@@ -100,8 +100,9 @@ def test_vector_distribution_get_samples_from_hypercube():
     initial_condition = distributions.VectorDistribution(values)
     result = initial_condition.get_samples_from_hypercube(30)
     assert len(result) == 30
-    for i in range(3):
-        assert 10 <= result.count(values[i]) <= 11
+    assert 10 <= result.count(values[0]) <= 11
+    assert 10 <= result.count(values[1]) <= 11
+    assert 10 <= result.count(values[2]) <= 11
```

Comment:
Does this have any advantage over the more compact version? Is it better style to write out the individual assertion statement?